src/__tests__/App.spec.tsx

```diff
@@ -28,4 +28,36 @@ test("loads and displays header", async () => {
   );
 
   expect(await screen.findByText("Posts")).toBeInTheDocument();
+  expect(await screen.findByText("Create a new post")).toBeInTheDocument();
+  expect(await screen.findByText("Delete post")).toBeInTheDocument();
+  expect(await screen.findByText("Update post")).toBeInTheDocument();
+});
+
+test("loads and displays post", async () => {
```

Comment:
How would you go about testing the other methods? (create, update, delete).
Please, provide detailed comments.